src/__tests__/posthog-core.identify.js

```diff
@@ -0,0 +1,285 @@
+import { PostHog } from '../posthog-core'
+import { PostHogPersistence } from '../posthog-persistence'
+
+jest.mock('../gdpr-utils', () => ({
+    ...jest.requireActual('../gdpr-utils'),
+    addOptOutCheck: (fn) => fn,
+}))
+jest.mock('../decide')
+
+given('lib', () => Object.assign(new PostHog(), given.overrides))
```

Comment:
most of these tests are copied almost as-is from posthog-core.js test file which was mahoosive